src/components/SiteHeader/SiteHeader.js

```diff
@@ -175,6 +175,11 @@ const SiteHeader = ({
                     element={Link}>
                     America by Design
                   </SideNavMenuItem>
+                  <SideNavMenuItem
+                    to="/impact/design-for-sustainability/"
+                    element={Link}>
+                    IBM Design for sustainability
+                  </SideNavMenuItem>
```

Comment:
This looks like this was only updated for the mobile nav and not desktop nav. URL is also `/practices/design-for-sustainability` it seems and not `/impact/design-for-sustainability`

Reply:
This needs to be removed since we moved the page under practices